src/slic3r/GUI/Tab.cpp

```diff
@@ -2156,8 +2156,8 @@ void TabPrint::build()
         optgroup->append_single_option_line("travel_jerk");
         
         optgroup = page->new_optgroup(L("Advanced"), L"param_advanced", 15);
-        optgroup->append_single_option_line("max_volumetric_extrusion_rate_slope");
-        optgroup->append_single_option_line("max_volumetric_extrusion_rate_slope_segment_length");
+        optgroup->append_single_option_line("max_volumetric_extrusion_rate_slope", "extrusion-rate-smoothing");
```

Comment:
@igiannakas

Reply:
Ah that’s how you do it! Perfect thank you, will know for next time! 👌🏻